eth/executionclient/config.go

```diff
@@ -8,7 +8,7 @@ import (
 
 // ExecutionOptions contains config configurations related to Ethereum execution client.
 type ExecutionOptions struct {
-	Addr                  string        `yaml:"ETH1Addr" env:"ETH_1_ADDR" env-required:"true" env-description:"Execution client WebSocket address"`
+	Addr                  string        `yaml:"ETH1Addr" env:"ETH_1_ADDR" env-required:"true" env-description:"Execution client WebSocket address. Supports multiple comma-separated addresses"`
```

Comment:
maybe just have `[]string` ? AFAIK viper knows how to parse this.

Reply:
Then it won't be backward compatible with the current values everyone uses, so it would require changing configs. Also, I think a significant amount of operators will continue using just one node, so a slice parameter might be less convenient

Reply:
```bash
% ./bin/ssvnode start-node 
2025/01/09 18:15:53 could not create logger could not read config: config file parsing error: yaml: unmarshal errors:
  line 14: cannot unmarshal !!str `ws://pr...` into []string
```